src/controllers/groupController.js

```diff
@@ -0,0 +1,40 @@
+import { DEFAULT_PAGE, PAGE_LIMIT } from '../config/global';
+import { GroupModel } from '../models/groupModel';
+import { UserModel } from '../models/userModel';
+import { UserGroupModel } from '../models/userGroupModel';
+import GroupService from '../services/groupService';
+
+const groupService = new GroupService(GroupModel, UserModel, UserGroupModel);
+
+const get = (req, res) => {
+    const { page = DEFAULT_PAGE } = req.params;
+    return groupService.getGroups(page, PAGE_LIMIT).then(data => res.json(data));
+};
+const getOne = (req, res) => groupService.getGroupById(req.params.id).then(user => res.json(user));
+const post = (req, res) => {
+    const data = req.body;
+    return groupService.addGroup(data).then(user => res.json(user));
```

Comment:
I might not see your entire logic and relationship between functions, however there might be no point in returning promises from your controller methods.
This is just a point for your consideration as a matter of consistency and transparency - it's not wrong in the slightest :)